contrib/images/umee.e2e.dockerfile

```diff
@@ -23,7 +26,6 @@ FROM ubuntu:rolling
 COPY --from=builder /go/pkg/mod/github.com/\!cosm\!wasm/wasmvm\@v*/internal/api/libwasmvm.*.so /usr/lib/
 COPY --from=builder /go/bin/* /usr/local/bin/
 COPY --from=builder /src/peggo-v*/peggo /usr/local/bin/
-RUN apt-get update && apt-get install ca-certificates -y
```

Comment:
We need this one, price-feeder try to get the price from api request `api/v1/prices`

Reply:
ah, ok, thank you!